specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProfile.json

```diff
@@ -1,395 +1,619 @@
-          "description": "Flag controlling whether the invoices for the billing profile are sent through email.",
-          "type": "boolean"
-        },
-        "poNumber": {
-          "description": "The purchase order name that will appear on the invoices generated for the billing profile.",
-          "type": "string"
-        }
-      }
+      "x-ms-azure-resource": true
     },
     "BillingProfileListResult": {
-      "description": "The list of billing profiles.",
+      "description": "A container for a list of resources",
+      "type": "object",
       "properties": {
         "nextLink": {
           "description": "The link (url) to the next page of results.",
-          "readOnly": true,
-          "type": "string"
-        },
-        "totalCount": {
-          "description": "Total number of records.",
-          "format": "int32",
-          "readOnly": true,
-          "type": "number"
+          "type": "string",
+          "readOnly": true
         },
         "value": {
-          "description": "The list of billing profiles.",
+          "description": "The list of resources.",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/BillingProfile"
           },
           "readOnly": true,
-          "type": "array"
+          "x-ms-identifiers": [
+            "id"
+          ]
         }
       }
     },
     "BillingProfileProperties": {
-      "description": "The properties of the billing profile.",
+      "description": "A billing profile.",
+      "type": "object",
       "properties": {
         "billingRelationshipType": {
           "description": "Identifies which services and purchases are paid by a billing profile.",
           "enum": [
+            "Unknown",
             "Direct",
             "IndirectCustomer",
             "IndirectPartner",
-            "CSPPartner"
+            "CSPPartner",
+            "CSPCustomer"
           ],
-          "readOnly": true,
           "type": "string",
+          "readOnly": true,
           "x-ms-enum": {
-            "modelAsString": true,
-            "name": "BillingRelationshipType"
+            "name": "BillingRelationshipType",
+            "modelAsString": true
           }
         },
         "billTo": {
-          "$ref": "./types.json#/definitions/AddressDetails",
-          "description": "Billing address."
+          "description": "Address details.",
+          "allOf": [
+            {
+              "$ref": "./types.json#/definitions/AddressDetails"
+            }
+          ]
         },
         "currency": {
           "description": "The currency in which the charges for the billing profile are billed.",
-          "readOnly": true,
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "displayName": {
-          "description": "The name of the billing profile.",
+          "description": "The name of the billing profile for the invoice section or customer.",
```

Comment:
Take a look at most recent swagger documentation to understand where this came from? Probably remove the addition?